geospaas_processing/cli/copy.py

```diff
@@ -37,6 +40,15 @@ def cli_parse_args():
     parser.add_argument(
         '-t', '--type', required=False, type=str,
         help="The type of dataset (as a str) which is written in flag file for further processing.")
+    parser.add_argument(
+        '-k', '--keeping_permanently', required=False, action='store_true',
```

Comment:
The long name of the argument is a bit confusing. There is no guarantee that the files will be kept permanently.

Reply:
Maybe this parameter is not necessary at all, you could probably make it work with the "obsoleteness" parameter alone.

Reply:
Yeah, that is the way I think first about it. We can set it to zero to dsable this ability of deletion. But @akorosov disagreed and requested for another cli input named `keep_permanently`. so both of us, please tell me to write the code with which approach? working with zero? or having another input like `keep_permanently`?

Reply:
discussed

Reply:
The part about the name is still valid though.